emulate_php_cli.c

```diff
@@ -0,0 +1,135 @@
+﻿#include <SAPI.h>
+#include <Zend/zend_alloc.h>
+#include <Zend/zend_exceptions.h>
+#include <Zend/zend_interfaces.h>
+#include <Zend/zend_types.h>
+#include <errno.h>
+#include <ext/spl/spl_exceptions.h>
+#include <ext/standard/head.h>
+#include <inttypes.h>
+#include <php.h>
+#include <php_config.h>
+#include <php_ini.h>
+#include <php_main.h>
+#include <php_output.h>
+#include <php_variables.h>
+#include <php_version.h>
```

Comment:
these are most definitely not all needed, but you can adjust that later @withinboredom